doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,209 @@
+    
+     - Can be either synchronous or asynchronous (i.e. `public void`, `public async ValueTask`, or `public async Task`)
+     - Any parameter that has a type and name (case-insensitive) matching a Feed or a State in this Model will be evaluated when the Command is invoked and its current latest value will be passed in as an argument.  
+        You'll find this feature to be very powerful in invoking commands by combining data from various Feeds in addition to a command parameter received from the View.
+
+        For example:
+
+        ```csharp
+        public IFeed<int> CounterValue => ...
+
+        public void ResetCounter(int counterValue)
+        {
+            ...
+        }
+        ```
+
+        When the command is executed, because the parameter name `counterValue` matches a feed name in the Model, this parameter will be materialized with the actual value from the Feed when this method is called on command execution.  
+        This behavior can be controlled and configured using the [`FeedParameter`](#feedparameter-attribute) and [`ImplicitFeedCommandParameter`](#implicitfeedcommandparameter-attribute) attributes.
+
+     - Can have one `CancellationToken` as its last parameter, but it's not mandatory.
+
+     > [!NOTE]  
+     > Name matching of command parameters to Feeds is case-insensitive.
+
+     > [!TIP]
+     > If you are using [Event Binding](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension#event-binding), you might need to opt out from command generation for those methods. See the next section on how to configure command generation.
+
+     ### Using attributes to control command generation
+
+     #### ImplicitCommands attribute
+
+     By default, implicit command generation is enabled by default. So that any method in the Model that matches the command criteria will be generated as a command.  
+     However, you may choose to switch this off for a specific method, class, or the entire assembly.
+
+     This can be achieved using the `ImplicitCommands` attribute. Here are some examples:
+
+     ```csharp
+     [assembly:ImplicitCommands(false)]
+     
+     [ImplicitCommands(true)]
+     public partial record MyModel();
+
+     [ImplicitCommands(true)]
+     public void MyCommand();
+     ```
+
+     These attributes can be used interchangeably to control command generation, for example, you may choose to disable the implicit feed command parameter altogether by using the attribute with the `assembly:` prefix. You can then opt-in for specific classes or methods. You can also switch off specific classes while opting-in certain methods by decorating them with this attribute as enabled. You can choose any combination to opt-in or out of implicit command generation.
+
+     #### FeedParameter attribute
+
+     You can explicitly match a parameter with a Feed even if the names don't match by decorating the parameter with the `FeedParameter` attribute:  
```

Comment:
```suggestion
     You can explicitly match a parameter with a Feed even if the names don't match. Do this by decorating the parameter with the `FeedParameter` attribute:  
```

Reply:
```suggestion
     You can explicitly match a parameter with a Feed even if the names don't match. This can be achieved by decorating the parameter with the `FeedParameter` attribute:  
```

Reply:
I don't know what (if any) style guides exist for the docs as a whole, but I do know that "This can be achieved" is an example of a phrase I've seen in guides as something _not_ to use. It's unspecific, undirected, and not encouraging action.